context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+This programming language is for people who want to create art or 
+images on the computer using code. 
 
 ## What is easy to do in this language? Why is it easy?
 
+Of course, it is very easy to create images. More specifically, it is easy to 
+create images with repeating components.  By defining rules, it is very easy to 
+create individual shapes which can be called and used in other parts of the image. 
+Furthermore, since users can call the rules in other parts of the code,
+recursion is relatively easy and shapes can be reused in different variations 
+again easily. Though saying it is easy to create images is very broad, what 
+comes to mind is how much cleaner it is to read in comparison to SVG, which is 
```

Comment:
I like the comparison here with SVG code. As you mention both are used to eventually make pictures, but ContextFree being a DSL is better at **describing** the picture where SVG is more like just an encoding of a picture. Personally, I found it much easier to translate code to picture versus the other way around.

I know with SVG's it is possible to name layers and groups so it has the potential to look more like ContextFree (except for the recursive part). Where in the spectrum of DSLs would you put SVGs out of curiosity?